axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -622,17 +643,22 @@ def test_strategy(self):
         self.transitions_test(state_and_actions)
 
         # finished: 0, 5, 10
-        state_and_actions = [(0, D), (12, D), (11, D)] + [(5, D), (10, C), (11, D), (5, D), (10, D), (8, C)] * 3
+        state_and_actions = [(0, D), (12, D), (11, D)] + [(5, D), (10, C),
+                                                          (11, D), (5, D),
+                                                          (10, D), (8, C)] * 3
         self.transitions_test(state_and_actions)
 
         # finished: 0, 2, 5, 10, 11, 12, 15
-        state_and_actions = ([(0, D), (12, C), (8, D), (5, D), (10, C), (11, C), (15, C), (15, C), (15, D)] +
+        state_and_actions = ([(0, D), (12, C), (8, D), (5, D),
+                              (10, C), (11, C), (15, C), (15, C), (15, D)] +
                              [(2, C)] * 3 + [(2, D), (14, C), (13, C)])
         self.transitions_test(state_and_actions)
 
         # finished: 0, 2, 3, 5, 10, 11, 12, 13, 14, 15
         to_state_fourteen = [(0, D), (12, D), (11, C), (15, D), (2, D)]
-        state_and_actions = to_state_fourteen + [(14, D), (13, C), (13, C), (13, D), (7, C)] + [(3, D), (3, C)] * 3
+        state_and_actions = to_state_fourteen + [(14, D), (13, C), (13, C),
+                                                 (13, D), (7, C)] + [(3, D),
+                                                                     (3, C)] * 3
```

Comment:
ha!  sorry.  (-_-*;)  i've been using what was set by pycharm, which is 120.  (apparently, there is some debate.  my brother was near frothing at the mouth giving me shit for sticking to 80 chars about a year back.)  after using 120 for a while, i agree with him, but i can absolutely make the effort to keep it at 80 here.